res/controllers/Traktor-Kontrol-S3-hid-scripts.js

```diff
@@ -1906,8 +1909,8 @@ TraktorS3.Controller.prototype.registerOutputPackets = function() {
     engine.connectControl("[Master]", "maximize_library", TraktorS3.bind(TraktorS3.Controller.prototype.maximizeLibraryOutput, this));
 
     // Master VuMeters
-    this.masterVuMeter["VuMeterL"].connection = engine.makeConnection("[Master]", "VuMeterL", TraktorS3.bind(TraktorS3.Controller.prototype.masterVuMeterHandler, this));
-    this.masterVuMeter["VuMeterR"].connection = engine.makeConnection("[Master]", "VuMeterR", TraktorS3.bind(TraktorS3.Controller.prototype.masterVuMeterHandler, this));
+    this.masterVuMeter.VuMeterL.connection = engine.makeUnbufferedConnection("[Master]", "VuMeterL", TraktorS3.bind(TraktorS3.Controller.prototype.masterVuMeterHandler, this));
```

Comment:
The mapping is broken in version 2.3 because this unrelated change slipped through.

https://bugs.launchpad.net/mixxx/+bug/1960680

Reply:
Sorry. I didnt see that this was a 2.3 PR
fix in #4671